app/lib/search/index_simple.dart

```diff
@@ -33,6 +33,21 @@ class SimplePackageIndex implements PackageIndex {
   bool get isReady => _isReady;
 
   @override
+  Future<bool> containsPackage(String package,
+      {String version, Duration maxAge}) async {
+    final String url = pubUrlOfPackage(package);
+    final PackageDocument doc = _documents[url];
+    if (doc == null) return false;
+    if (version != null && doc.version != version) return false;
+    if (maxAge != null &&
+        (doc.timestamp == null ||
+            new DateTime.now().difference(doc.timestamp) > maxAge)) {
```

Comment:
Is `doc.timestamp` utc or not? We should use the same time zone.

Reply:
good catch, fixing